dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/ProcessUtils.java

```diff
@@ -352,13 +342,13 @@ public static void kill(TaskExecutionContext taskExecutionContext) {
   }
 
   /**
-   * get pids str
+   * get pids str.
    *
    * @param processId process id
    * @return pids
    * @throws Exception exception
    */
-  public static String getPidsStr(int processId)throws Exception{
+  public static String getPidsStr(int processId)throws Exception {
```

Comment:
```suggestion
  public static String getPidsStr(int processId) throws Exception {
```

Reply:
OK，I'll change it. At present, ckckstyle has not been detected. I hope it can be added.

Reply:
> OK，I'll change it. At present, ckckstyle has not been detected. I hope it can be added.

Ok, Thx for your reminding~